src/test/java/org/fisco/bcos/channel/test/parallel/precompile/PerformanceDTTest.java

```diff
@@ -286,44 +291,141 @@
         }
     }
 
+    public void userTransferSignTxPerfTest(BigInteger totalSignedTxCount, int threadC)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'userTransferSignTxPerfTest(BigInteger, int)' has an NPath complexity of 256](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4981260)